auacm/app/static/js/modules/mainModule.js

```diff
@@ -54,6 +54,21 @@ app.config(function($routeProvider) {
             templateUrl : 'static/html/settings.html',
             controller : 'SettingsController',
             activetab : 'settings'
+        })
+        .when('/problems/create', { // route for problem creation page
+            templateUrl : 'static/html/createProblem.html',
+            controller : 'CreateProblemController',
+            activetab  : 'settings'
+        })
+        .when('/problems/:pid', { // route for the problem view page
+            templateUrl : 'static/html/viewProblem.html',
+            controller : 'ViewProblemController',
+            activetab : 'problems'
+        })
+        .when('/problems/edit/:pid', { // route for problem edit page
```

Comment:
I know this is a little pedantic, but let's do 

    .when('/problems/:pid/edit', ...

to mimic more of a RESTful interface.